bin/install

```diff
@@ -17,13 +17,16 @@ use strict;
 use File::Basename qw(basename dirname);
 use Getopt::Std qw(getopts);
 
-my ($VERSION) = '1.3';
+our $VERSION = '1.3';
```

Comment:
Update version now that a feature has been removed

Reply:
Sure, committed.